java/ql/src/Security/CWE/CWE-094/ArbitraryAPKInstallation.ql

```diff
@@ -0,0 +1,202 @@
+/**
+ * @id java/android/arbitrary-apk-installation
+ * @name Android APK installation
+ * @description Creating an intent with a URI pointing to a untrusted file can lead to the installation of an untrusted application.
+ * @kind path-problem
+ * @security-severity 9.3
+ * @problem.severity warning
+ * @precision medium
+ * @tags security
+ *       external/cwe/cwe-094
+ */
+
+import java
+import semmle.code.java.frameworks.android.Intent
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.TaintTracking2
+import semmle.code.java.dataflow.TaintTracking3
```

Comment:
Now that https://github.com/github/codeql/pull/12186 has been merged, I think it would be a good idea to use the new configuration modules, instead of these configuration class copies. Let me know if a pairing session would be useful for the rewrite.

/cc @jcogs33 as well.